### PR TITLE
Post tweet without images

### LINKED ISSUE #36: Allow posting feedback with no images

### PR DESCRIPTION
**Pull request by shachar-langer**
Closing #36

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -63,7 +64,9 @@
     "@types/twit": "2.2.28",
     "chai": "3.4.1",
     "chai-http": "4.3.0",
+    "http-shutdown": "^1.2.2",
```
Comment (will-weiss) [inline package.json]: get rid of this

File: package.json [hunk 2]
```diff
@@ -43,6 +43,7 @@
     "@types/cheerio": "0.22.22",
     "@types/dotenv": "8.2.0",
     "@types/knex": "0.16.1",
+    "@types/knex-cleaner": "^1.3.0",
```
Comment (will-weiss) [inline package.json]: hard code versions

File: package.json (hunk 1 above)
Comment (shachar-langer) [inline package.json]: remove

File: src/__test__/integration/postTweet.test.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  it('should return a valid response for a tweet without images', async () => {
+    expect(await agent.post('/v1/auth/fake')).to.have.property('status', 200)
+    const response = await agent
+      .post('/v1/feedback')
+      .send({ host: 'morehumaninternet.org', status: 'Make the internet great again' })
+      .expect(201, { url: "https://twitter.com/@morehumaninter1/statuses/1111111" })
+
+    expect(response.body).to.eql({ url: "https://twitter.com/@morehumaninter1/statuses/1111111" })
+
+    // TODO - test that the database has the website and the tweet
```
Comment (shachar-langer) [inline src/__test__/integration/postTweet.test.ts]: TODO

File: src/handlers.ts [hunk 4]
```diff
@@ -77,16 +77,20 @@ export async function getWebsite(ctx: IRouterContext): Promise<any> {
 // Redirect the user to Twitter for authentication. When complete, Twitter
 // will redirect the user back to the application at
 // /auth/twitter/callback
-export const authTwitter = passport.authenticate('twitter')
+export const authTwitter = () => {
```
Comment (shachar-langer) [inline src/handlers.ts]: one liner

File: \ [hunk 5]
```diff
@@ -0,0 +1,26 @@
+Add tests that hang
```
Comment (shachar-langer) [inline \]: Remove

**Review by will-weiss — APPROVED** [review]

**Pull request merged**